docker/datahub-mce-consumer/Dockerfile

```diff
@@ -30,9 +30,10 @@ USER datahub
 
 ENV JMX_OPTS=""
 ENV JAVA_OPTS=""
+ENV ENABLE_PROMETHEUS="true"
```

Comment:
We should be able to set this at the `helm` yaml level right? instead of modifying the datahub docker image?